pkg/fieldpath/paved.go

```diff
@@ -204,6 +204,8 @@ func expandWildcards(data any, segments Segments) ([]Segments, error) { //nolint
 					}
 					res = append(res, r...)
 				}
+			case nil:
```

Comment:
Let's add a new test case here similar to the following:
```go
			path:   "spec.containers[*].a",
			data:   []byte(`{"spec":{"containers":null}}`),
```

Reply:
Added. Thanks @ulucinar